documentation/API_documentation/QualityByDesign_User_Story.md

```diff
@@ -0,0 +1,15 @@
+# Quality-By-Design (QbD) User Story
+
+| **Item**                        | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **_Summary_**                   | As an application developer, I want to be able to share network KPI requirements with a Communication Service Provider (CSP) and stream these KPIs for real-time monitoring. This allows the CSP to track network performance and ensure it aligns with the specified requirements. If any deviations occur, the CSP can proactively suggest or implement corrective measures to maintain optimal performance.                                                                                                                      |
```

Comment:
I think the first sentence could be rephrased to highlight the difference with Connectivity Insights (which also involves sharing of KPIs): so suggest to focus on the stream aspect, e.g. 

"As an application developer, I want to be able to stream my application's network KPI requirements to the network operator. This allows the network operator to monitor in real-time whether it can fulfil these dynamic requirements, and if not, proactively suggest or implement corrective measures to meet the application's requirements."

(also in the above I have suggested to change 'CSP' to 'network operator' as it's common in CAMARA, and 'maintain optimal performance' to 'meet application requirements', since we don't know whether performance was optimal in the first place, and I don't believe we are targeting optimal so much as that which the application developer and end user are happy with, or better.

Reply:
thanks Kevin, good feedback! I've incorporated and will update the PR soon.